src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/TurnoverPolicyController.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.companieshouse.web.accounts.controller.smallfull;
+
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import uk.gov.companieshouse.web.accounts.annotation.NextController;
+import uk.gov.companieshouse.web.accounts.annotation.PreviousController;
+import uk.gov.companieshouse.web.accounts.controller.BaseController;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.accountingpolicies.TurnoverPolicy;
+import uk.gov.companieshouse.web.accounts.service.smallfull.TurnoverPolicyService;
+import uk.gov.companieshouse.web.accounts.util.Navigator;
+import uk.gov.companieshouse.web.accounts.validation.ValidationError;
+
+@Controller
+@NextController(ReviewController.class)
+@PreviousController(BasisOfPreparationController.class)
+@RequestMapping("/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/small-full/turnover-policy")
+public class TurnoverPolicyController extends BaseController {
+
+    @Autowired
+    private TurnoverPolicyService turnoverPolicyService;
+
+    @GetMapping
+    public String getTurnoverPolicy(@PathVariable String companyNumber,
+        @PathVariable String transactionId,
```

Comment:
Not important, but can we align the parameters of this method so the `@PathVariable`'s sit under one another? Makes for easier readability.

Reply:
I've used the intellij formatting which follows our coding.